app/views/staff/activity_forms/purpose.html.haml

```diff
@@ -1,5 +1,5 @@
 = render layout: "wrapper" do |f|
   %h1.govuk-heading-xl= t("helpers.fieldset.activity.purpose", level: t("page_content.activity.level.#{f.object.level}"))
 
-  = f.govuk_text_field :title
+  = f.govuk_text_field :title, label: { text: I18n.t("helpers.fieldset.activity.title", level: t("page_content.activity.level.#{f.object.level}").titlecase) }
```

Comment:
One of my issues with all this is how to know what keys to use, I would expect this to be under `helpers.label.activity.title` but it isn't, without a robust way to organise these they just seem to proliferate and it becomes hard to know which key belongs to which context.